assets/prefabs/CookingRecipes.prefab

```diff
@@ -248,6 +248,90 @@
         },
         "outputCount": 1
       }
+      "Cooking:BoiledEgg": {
+        "inputs": {
+          "Cooking:Egg": 1
+        },
+        "outputCount": 1
+      }
+      "Cooking:CookedBean": {
+        "inputs": {
+          "Cooking:bean":1,
+          "Cooking:onion":1,
+          "Cooking:tomato":1,
+          "Cooking:salt":1
+        },
+        "outputCount": 1
+      }
+      "CookedBrownRice": {
+        "inputs": {
+          "Cooking:BrownRice": 1
+        },
+        "outputCount": 1
+      }
+      "CookedCorn": {
+        "inputs": {
+          "Cooking:maize": 1
+        },
+        "outputCount": 1
+      }
+      "cookedKale": {
+        "inputs": {
+          "Cooking:kale": 1
+        },
+        "outputCount": 1
+      }
+      CookedRice: {
+        "inputs": {
+          "Cooking:rice": 1
```

Comment:
According to https://github.com/Terasology/Cooking/blob/develop/assets/prefabs/recipe/cook/RecipeCookedRice.prefab#L22, this needs to be `whiteRice`